pkg/gpu/testdata/cudasample.c

```diff
@@ -30,14 +30,20 @@ cudaError_t cudaStreamSynchronize(cudaStream_t stream) {
 int main(int argc, char **argv) {
     cudaStream_t stream = 30;
 
+    fprintf(stderr, "Running CUDA sample. Waiting to allow uprobes to attach...\n");
+
     // Give time for the eBPF program to load
     sleep(5);
```

Comment:
_nit:_ maybe change that to use the value from the argv in case we need to toggle it

Reply:
I'll have that change in the multi-gpu PR, as I'll need to send some parameters from the test.